tests/prefetch_related/tests.py

```diff
@@ -199,23 +199,25 @@ def test_reverse_one_to_one_then_m2m(self):
 
     def test_attribute_error(self):
         qs = Reader.objects.all().prefetch_related('books_read__xyz')
-        with self.assertRaises(AttributeError) as cm:
+        msg = "Cannot find 'xyz' on Book object, 'books_read__xyz' is an invalid parameter to prefetch_related()"
+        with self.assertRaisesMessage(AttributeError, msg) as cm:
             list(qs)
 
         self.assertIn('prefetch_related', str(cm.exception))
 
     def test_invalid_final_lookup(self):
         qs = Book.objects.prefetch_related('authors__name')
-        with self.assertRaises(ValueError) as cm:
+        msg = "'authors__name' does not resolve to an item that supports prefetching - " \
+              "this is an invalid parameter to prefetch_related()."
+        with self.assertRaisesMessage(ValueError, msg) as cm:
             list(qs)
 
         self.assertIn('prefetch_related', str(cm.exception))
         self.assertIn("name", str(cm.exception))
 
     def test_forward_m2m_to_attr_conflict(self):
-        msg = 'to_attr=authors conflicts with a field on the Book model.'
```

Comment:
I don't see a big advantage to this change. The coding style says to use longer lines if it makes things easier to read -- my taste is to use `msg = '...'` if `with self.assertRaisesMessage(ValueError, '....'):.` is much over 79 chars.